test/e2e/play_kube_test.go

```diff
@@ -3648,7 +3648,7 @@ o: {{ .Options.o }}`})
 		Expect(inspect.OutputToString()).To(ContainSubstring("o: " + volOpts))
 	})
 
-	It("podman play kube persistentVolumeClaim with source", func() {
+	FIt("podman play kube persistentVolumeClaim with source", func() {
```

Comment:
somebody is not using my new `FOCUS=...` Makefile option :(

Reply:
A good reminder to change my muscle memory and bash history